cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -217,7 +217,9 @@ public void create(String namespaceId, String name, DatasetInstanceConfiguration
     DatasetSpecification spec = opExecutorClient.create(newInstance, typeMeta,
                                                         DatasetProperties.builder()
                                                           .addAll(props.getProperties())
-                                                          .build());
```

Comment:
The configure method can still add a description if it wants, right? It will just get overwritten if user specifies description during instance creation.